pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "metasynth"
+authors = [
+    {name = "Raoul Schram", email = "r.d.schram@uu.nl"},
+    {name = "Erik-Jan van Kesteren", email = "e.vankesteren1@uu.nl"},
+]
+description = "Package for creating synthetic datasets while preserving privacy."
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["metadata", "open-data", "privacy", "synthetic-data", "tabular datasets"]
+license = {text = "MIT"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+	"pandas",
+	"polars>=0.14.17",
+	"numpy>=1.20",
+    "pyarrow",  # Dependency of polars since we're converting from pandas.
+    "scipy",
+    "numpy>=1.20",
+    "faker",
+    "scikit-learn",
+    "jsonschema",
+    "importlib-metadata;python_version<'3.10'",
```

Comment:
Does this mean we need to change this when the python version goes above 3.9?

Reply:
Yes, when Python 3.9 becomes unsupported, we can remove the line.